nav-app/src/assets/config.json

```diff
@@ -98,6 +98,7 @@
     },
 
     "comment_color": "yellow",
+    "show_comment_underline": true,
```

Comment:
I think this should probably be a property of the `features` array. That way it can be customized in the _create a customized Navigator_ interface. Otherwise you could already get this behavior by setting `comment_color` to `"transparent"`.